docs/maintainer/knowledge_base.md

```diff
@@ -1480,6 +1480,31 @@ In order to qualify as a noarch python package, all of the following criteria mu
   in the `build` section of `meta.yaml`
 - No activate scripts
 
+All recipes employing `noarch: python` should use the `python_min` variable per the following example:
+
+```yaml title="recipe/meta.yaml"
+name: package
```

Comment:
```suggestion
{% set python_min = python_min|default("0.1a0") %}

name: package
```

Reply:
Could you elaborate on why this needs to be set? The CFEP says this comes from the global pinning

https://github.com/conda-forge/cfep/blob/main/cfep-25.md#specification

Reply:
This is a default that is needed for recipe parsing, but is eventually overridden by the default from the pinnings. TBH this is a workaround for a bug in conda-build, but I have not been able to track it down.

Reply:
Thanks for the info

Reply:
No longer seems to be needed https://github.com/conda-forge/conda-forge.github.io/pull/2360/files#diff-f233d562a21155b034d2aa5f9d8a8148cc0952b24c34f8e0e6cf0a1a4e241acfL1486-L1487